### PR TITLE
Kb behavior

### PR DESCRIPTION
**Pull request by keithbradley1**
here are the new tests for behavior predictor

### CODE REVIEW

File: behavior_predictor_lexicon.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+behavior_predictor_lexicon =
+  {
+    "dictator": [("commander": .08), ("authoritarian": .08)],
```
Comment (carter-harris) [inline behavior_predictor_lexicon.py]: no ":" between the tuples info, "," comma dudes instead

File: behavior_predictor_lexicon.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+behavior_predictor_lexicon =
+  {
+    "dictator": [("commander", .08), ("authoritarian", .08)],
+    "hippie": [("freethinker", .08), ("bohemian", .08)],
+    "blame": [("you", .08), ("they", .08, ("charge", .08), ("censure", .08)],
```
Comment (carter-harris) [inline behavior_predictor_lexicon.py:5]: missing a ")" on the second tuple of this line

File: behavior_predictor_lexicon.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "inspiring": [("sway", .08), ("invigorate", .08), ("produce", .08)],
+    "hate": [("antagonism", .08), ("abomination", .08), ("grievance", .08)],
+    "childish": [("foolish", .08), ("naive", .08)],
+    "mockery": [("smart", .08), ("farse", .08), ("sham", 08)],
+    "pandering": [("smart", .08), ("please", .08), ("satisfy", 08)],
+    "supportive": [("good", .08), ("backing", .08), ("footing", 08)],
+    "promises": [("commitment", .08), ("obligation", .08)],
```
Comment (carter-harris) [inline behavior_predictor_lexicon.py:12]: missing a ")" on the second tuple of this line

File: behavior_predictor_lexicon.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    "patriotic": [("country", .08), ("national", .08), ("loyal", 08)]
```
Comment (Jessicashinjo) [inline behavior_predictor_lexicon.py]: Missing period infront of the 08 integers on lines 9-11 and line 13.

File: test_behavior_predictor.py [hunk 5]
```diff
@@ -4,180 +4,120 @@
 #################### behavior predictor tests ####################
 
 class TestBehavior(unittest.TestCase):
-''' Tests that behavoir predictor receives a message as a list to be evaluated'''
-  pass
+''' Tests that behavoir predictor receives a message to be evaluated'''
```
Comment (Jessicashinjo) [inline test_behavior_predictor.py]: You might want to correct the spelling from behavoir to behavior.

**Comment by Jessicashinjo** [comment]
All the changes look good 👍

**Pull request merged**